docs/ecosystem/ifttt/readme.md

```diff
@@ -11,71 +11,48 @@ meta:
-        <img alt="" src="./images/ifttt2.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+Before you start creating IFTTT applets you need to make sure that you configured your local openHAB runtime to expose certain items to myopenHAB.org. Only those items will be visible to IFTTT. Read here [how to expose items to myopenHAB.org](https://www.openhab.org/addons/integrations/openhabcloud/#configuration). You will also be able to send commands to those items from IFTTT Applets. 
 
-  <div class="da-animate da-slideFromLeft" style="display: block;">
-    <span class="iconWrapper iconLink icon-search" style="font-style: italic; margin-top: 63px"></span>
-  </div></a>
+**Important:** Items will appear in myopenHAB and thus in IFTTT only after at least one state update has been received by myopenHAB.org from your local openHAB runtime. 
 
-  <h3>Open my garage when I turn on the ignition</h3>
-  </article>
+This can be done by for instance creating a simple rule:
 
-  <article class="span4 bloc">
-    <a href="https://ifttt.com/recipes/299083-announce-finished-tasks-using-tts">
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt3.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+```
+rule "Set initial values IFTTT items"
+when
+	System started
+then
+	createTimer(now.plusSeconds(160)) [|
+		logInfo("RULE", "Set initial values IFTTT items")
+		postUpdate(YourItem, 0)
+	]
+end
+```
 
-  <div class="da-animate da-slideFromRight" style="display: block;">
-    <span class="iconWrapper iconLink icon-search" style="font-style: italic; margin-top: 63px"></span>
-  </div></a>
+**Sending notifications from openHAB to your mobile device via IFTTT**
 
-  <h3>Announce finished tasks through text-to-speech</h3>
-  </article>
-</div>
-
-Besides the new kinds of integration possibilities, IFTTT is also a perfect start for new openHAB users as it makes the creation of rules dead simple - no need to do any scripting, no complex setup routines, it all works through a few mouse clicks!
-
-<div class="img-wrapper"><img src='./images/ifttt-do.jpg' alt='IFTTT Do Button'/></div>
-
-We have also integrated with IFTTT’s [Do Button](https://ifttt.com/products/do/button). The Do button allows you to access often required functionality very quickly - either through the Do app, as a widget or even as a floating button, that is always present on your screen.
-
-## Let's Go!
-
-IFTTT is available to all our users through the [myopenHAB service](http://www.myopenhab.org/). Once you have your myopenHAB account set up, all you have to do is to decide, which items you want to share with IFTTT and to activate the openHAB channel.
+If you are using iOS or Android, you can use IFTTT to send notifications from your openHAB runtime to your device running iOS or Android. Read more about this [here](https://community.openhab.org/t/openhab-send-sensor-notification-to-ios-android-using-ifttt/24725).
```

Comment:
Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.